apps/vm-lang/main.cpp

```diff
@@ -329,5 +329,10 @@ int main(int argc, char **argv)
     state_map.SaveToFile(data_path.c_str());
   }
 
+  if (!success)
+  {
+    std::cout << error << std::endl;
```

Comment:
Shouldn't this be

```suggestion
    std::cerr << error << std::endl;
```

?

Reply:
I am sure that I made this change to `std::cerr` previously? 😕